cou/utils/nova_compute.py

```diff
@@ -36,7 +37,20 @@ async def get_empty_hypervisors(units: list[Unit], model: Model) -> list[Machine
     tasks = [get_instance_count(unit.name, model) for unit in units]
     instances = await asyncio.gather(*tasks)
     units_instances = zip(units, instances)
-    return [unit.machine for unit, instances in units_instances if instances == 0]
+    empty_units = set()
+    empty_machines = []
+
+    for unit, instance_count in units_instances:
+        if instance_count == 0:
+            empty_units.add(unit)
+            empty_machines.append(unit.machine)
+
+    skipped_units = set(units) - empty_units
+
+    logger.info("Found non-empty hypervisors: %s", stringify_units(skipped_units))
```

Comment:
I think this might be a warning log. I think it's good to show it by default to the users

Reply:
There's also the issue that this is going to happen even that there are no empty_units. I'm not a huge fan of this, but we might need to add a if condition to log it:

```python
if skipped_units:
    logger.warning(....)
```